pandas/tests/extension/base/ops.py

```diff
@@ -263,3 +263,10 @@ def test_unary_ufunc_dunder_equivalence(self, data, ufunc):
         else:
             alt = ufunc(data)
             tm.assert_extension_array_equal(result, alt)
+
+
+class BaseRoundingTests:
+    def test_round(self, data):
+        result = pd.Series(data).round()
+        expected = pd.Series(np.round(data), dtype=data.dtype)
```

Comment:
To be more generic, this should probably be defined to call `[round(element) for element in data]`

Reply:
done